### PR TITLE
Created the Nav bar for the front page

### PR DESCRIPTION
**Pull request by AbdussamiG**
The first html file is just for the navbar of the front page.

### CODE REVIEW

**Comment by AbdussamiG** [comment]
Okay I’ll do that

**Comment by Frost199** [comment]
I just saw your folder structure well, just remove the folder navbar, i isn't necessary, I noticed it is in a folder called Fron-page, the only change you need to make is the renaming your folders to small letters

**Comment by AbdussamiG** [comment]
Done, I hope its okay now?

File: Pages/front-page/css-style/styles.css [hunk 1]
```diff
@@ -0,0 +1,68 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@500&display=swap');
```
Comment (tonymj76) [inline Pages/front-page/css-style/styles.css:1]: please pull from the recent branch and modify your changes

File: Pages/front-page/index.html [hunk 2]
```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
```
Comment (tonymj76) [inline Pages/front-page/index.html:1]: we are using ejs not vanilla html

**Comment by timothymayor** [comment]
Please, kindly you push to the develop branch not master!

**Comment by AbdussamiG** [comment]
Alright I’ll do that

**Pull request closed without merging**